gnarly.rb

```diff
@@ -35,10 +35,9 @@ def create_gnarly_rails_app
 
 def add_gems
   gem_group :development, :test do
-    gem 'axe-matchers'
+    gem 'axe-core-rspec'
```

Comment:
I [think](https://github.com/KevinBongart/automated_accessibility_testing_with_ruby_on_rails#step-3) we also need `axe-core-capybara` - but I've only read that, not implemented it myself.

Reply:
#159